mmv1/templates/terraform/custom_import/apigee_env_addons.go.tmpl

```diff
@@ -0,0 +1,13 @@
+config := meta.(*transport_tpg.Config)
+
+// current import_formats cannot import fields with forward slashes in their value
+if err := tpgresource.ParseImportId([]string{"(?P<env_id>.+)/addonsConfig"}, d, config); err != nil {
+	return nil, err
+}
+
+nameParts := strings.Split(d.Get("env_id").(string), "/")
+if len(nameParts) == 5 {
+	id := fmt.Sprintf("organizations/%s/environments/%s", nameParts[1], nameParts[3])
+	d.SetId(id)
+}
```

Comment:
Should be something like this:

```suggestion
id := d.Get("env_id").(string)
nameParts := strings.Split(id, "/")
if len(nameParts) != 4 {  // you could be stricter about this check if you want.
        return nil, fmt.Errorf("env is expected to have shape organizations/{{"{{"}}org_id{{"}}"}}/environments/{{"{{"}}env{{"}}"}}, got %s instead", id)
}
d.SetId(id)
```

Any errors parsing the import id need to be returned - otherwise you might end up with an import that looks like it succeeds but fails to actually set an id (which means the resource won't actually be imported).